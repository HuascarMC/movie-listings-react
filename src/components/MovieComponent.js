import React from 'react';
import axios from 'axios'

class MovieComponent extends React.Component {

  render() {
    return(
      <div>
      </div>
    );
  }
}

export default MovieComponent;
