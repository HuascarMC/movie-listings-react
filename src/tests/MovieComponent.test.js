import React from 'react';
import MovieComponent from '../components/MovieComponent'
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

it('should render without crashing', () => {
  shallow(<MovieComponent />);
});

it('should render div with movie-component class', () => {
  const wrapper = shallow(<MovieComponent />);
  const div = wrapper.find('.movie-component')
  expect(div.length).toEqual(1);
});

it('should have div with style property', () => {
  const wrapper = shallow(<MovieComponent />);
  const div = wrapper.find('.movie-component')
  expect(div.prop('style')).toBeTruthy();
});
