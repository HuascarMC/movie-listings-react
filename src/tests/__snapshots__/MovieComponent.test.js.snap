// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="movie-component"
  style={
    Object {
      "backgroundImage": "url()",
    }
  }
>
  <div
    className="preview-info"
  >
    <ul>
      <li
        className="movie-type"
      >
        
      </li>
      <li
        className="movie-title"
      >
        
      </li>
    </ul>
  </div>
</div>
`;
